Framework/Nexus/test/NeXusFileTest.h

```diff
@@ -39,16 +41,17 @@ class NeXusFileTest : public CxxTest::TestSuite {
   static NeXusFileTest *createSuite() { return new NeXusFileTest(); }
   static void destroySuite(NeXusFileTest *suite) { delete suite; }
 
-  void test_compile() {
-    string x = strmakef("not_a_real_file_%d", 10);
-    removeFile(x);
-    TS_ASSERT(true);
-  }
+  // void setUp() { mkdir( "playground" ); }
+  // void tearDown() { system( "rm -Rf playground"); }
+
+  // #################################################################################################################
+  // TEST CONSTRUCTORS
+  // #################################################################################################################
 
   void test_remove() {
     // create a simple file, and make sure removeFile works as intended
     cout << "\nremoving\n";
-    string filename("not_a_real_file.txt");
```

Comment:
Can he tests be refactored so `FileResource` stays as a variable in scope? Then the test would automatically cleanup after itself and `test_remove()` would not be needed anymore.